### PR TITLE
Update to NodeCG 0.9.4

### PR DESCRIPTION
**Pull request by TreZc0**
Updated the files to match the new update, added a few new extra tools and dashboard panels and moved the 0.8. readme to older versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,12 @@ Check the [`old-versions`](https://github.com/nodecg/awesome-nodecg/blob/master/
 NodeCG is a broadcast graphics framework and application. It provides an API and structure for making dynamic graphical assets for broadcast use. NodeCG graphics packages are called "bundles". NodeCG is primarily aimed at [Twitch](http://www.twitch.tv) broadcasters using [Open Broadcaster Software](https://obsproject.com), but is usable in any environment that can render HTML, including [CasparCG](http://casparcg.com).
 
 Still not clear on what NodeCG is and what it is used for? These videos go over some of the more prominent uses of NodeCG:
-- [Tip of the Hats 2015 Graphics Overview](https://www.youtube.com/watch?v=Z4-qaya5-0Y)
 - [Tip of the Hats 2016 Graphics Overview](https://www.youtube.com/watch?v=F0xGN-aSytA)
 - [Awesome Games Done Quick 2016 Graphics Overview](https://www.youtube.com/watch?v=oAzj9Zddogs)
```
Comment (Lange) [inline README.md]: I think the AGDQ2016 overview should be removed, as it tends to raise more questions than it answers due to its unusual use of Electron as the renderer.

File: README.md [hunk 2]
```diff
@@ -28,11 +27,9 @@ Want to suggest a bundle for inclusion? Open a PR!
 - [Dashboard Polymer Elements](#dashboard-polymer-elements)
 
 ## Complete Systems
-NodeCG bundles (or suites of bundles) that provide an entire production system by themselves. They're generally targeted at a specific game or type of show, though some may be more generalized.
+NodeCG bundles (or suites of bundles) that provide an entire production system by themselves. They're generally targeted at a specific game or type of show, though some may be more generalized. Right now, there is only a single published Bundle for NodeCG 0.9:
```
Comment (Lange) [inline README.md]: "Bundle" should be lowercase: `bundle`

File: README.md [hunk 3]
```diff
@@ -56,9 +53,13 @@ Bundles which provide some form of visual information on an overlay.
 
 ## Utilities
 - [prime-manual-alerts](https://github.com/ProbablePrime/prime-manual-alerts) - Manual emits follow, subscription, host and tip events which prime-alerts can listen to.
+- [prime-socialmedia](https://github.com/ProbablePrime/prime-socialmedia) - A NodeCG social media displayer for popups with information on several social media platforms.
+- [streen](https://github.com/SupportClass/streen) - a centralized interface for interacting with Twitch Chat.
+- [Dashboard](https://github.com/nodecg/dashboard) - A Standalone application to check and use your NodeCG Dashboard.
+- [NodeCG-OBS](https://github.com/nodecg/nodecg-obs) - A OBS-Websocket bower module to connect to multiple OBS instances at the same time.
```
Comment (Lange) [inline README.md]: Change description to `adds the ability to connect to 1-4 instances of OBS (via obs-websocket) to your NodeCG bundle`

Comment (TreZc0) [reply README.md]: Sounds good to me. I am not sure if there are more modules to add, i browsed some common githubs i knew from messing with my layouts, but didn't find a lot more.

**Pull request merged**